tests/dummy/app/templates/docs/testing-acceptance.md

```diff
@@ -1,16 +1,35 @@
 # Acceptance Testing
 
-## Standalone Engine
+## Testing Standalone Engine
```

Comment:
Testing seems obvious since this is in a section called “acceptance testing”

Reply:
I’m inspiring in the official guides https://guides.emberjs.com/release/testing/testing-application/

Reply:
Testing > Testing Components > Testing User Interaction

Reply:
Testing > Testing Components > Testing User Interaction